### PR TITLE
error display for loginform added

### PR DESCRIPTION
**Pull request by karthijey**
@bbmoz Changes added as per the design 151586060.Please review the login form. Thanks.

### CODE REVIEW

File: client/src/components/LoginForm/LoginForm.scss [hunk 1]
```diff
@@ -40,3 +40,16 @@ h1 {
     color: $primaryColorHover;
   }
 }
+
+.buttonError {
+  @extend %buttonErr;
+  @extend %field;
```
Comment (bbmoz) [inline client/src/components/LoginForm/LoginForm.scss]: `%field` was removed as part of this PR: https://github.com/CodeForSocialGood/calltocode.org/commit/0a4b0026bf932ce78fd20e35cfa7b8733f7e9505

File: server/controllers/loginController.js [hunk 2]
```diff
@@ -16,6 +16,12 @@ const loginController = {
 
       if (user && password === user.password) {
         return res.sendStatus(200)
+      } else if (user && password !== user.password) {
+        res.statusMessage = 'Wrong Password'
+      }
+
+      if (user === null) {
+        res.statusMessage = 'Wrong Email'
```
Comment (bbmoz) [inline server/controllers/loginController.js]: Should this be `res.statusText` instead?

File: client/src/components/LoginForm/LoginForm.js [hunk 3]
```diff
@@ -32,14 +32,23 @@ class LoginForm extends Component {
...
+      if (response.statusText === 'Wrong Email') {
+        throw new SubmissionError({ email: response.statusText,
+          _error: 'Incorrect credentials, please try again!!' })
+      } else if (response.statusText === 'Wrong Password') {
+        throw new SubmissionError({ password: response.statusText,
+          _error: 'Incorrect credentials, please try again!' })
+      } else {
+        throw new SubmissionError({ email,
+          _error: 'Incorrect credentials, please try again!' })
+      }
```
Comment (bbmoz) [inline client/src/components/LoginForm/LoginForm.js]: I think you can clean this up by doing something like:
```js
const _error = 'Incorrect credentials, please try again!'
if (response.statusText === 'Wrong Email') {
  throw new SubmissionError({
    email: response.statusText,
    _error
  })
} else if...
```

File: client/src/components/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ class LoginForm extends Component {
       <input className={styles.inputPassword}
         placeholder="Password"
         type="password"
-        {...field.input} />
+        {...field.input} title = {field.meta.error} />
```
Comment (bbmoz) [inline client/src/components/LoginForm/LoginForm.js]: What is the `title` used for here?

File: client/src/components/LoginForm/LoginForm.js [hunk 5]
```diff
@@ -32,14 +32,23 @@ class LoginForm extends Component {
...
+          _error: 'Incorrect credentials, please try again!' })
+      }
     } else {
       this.props.login({ email })
     }
   }
 
   render () {
-    const { handleSubmit } = this.props
+    const { handleSubmit, error } = this.props
```
Comment (bbmoz) [inline client/src/components/LoginForm/LoginForm.js]: I assume `error` is coming from the `_error` in SubmissionError. Is that right?

File: client/src/components/LoginForm/LoginForm.js (hunk 4 above)
Comment (bbmoz) [inline client/src/components/LoginForm/LoginForm.js]: Add a new line for each attribute, like this:
```js
<input className={styles.inputPassword}
  placeholder="Password"
  type="password"
  title = {field.meta.error}
  {...field.input} />
```

Comment (karthijey) [reply client/src/components/LoginForm/LoginForm.js]: It's just to show the error.

File: client/src/components/LoginForm/LoginForm.js (hunk 5 above)
Comment (karthijey) [reply client/src/components/LoginForm/LoginForm.js]: Yes, it is from the SubmissionError.

**Pull request merged**